tools/reviewer/local_server/service/CodeReviewService.java

```diff
@@ -376,6 +377,22 @@ public void ping(Empty req, StreamObserver<PongResponse> responseObserver) {
     responseObserver.onCompleted();
   }
 
+  @Override
+  public void removeWorkspace(RemoveWorkspaceRequest req, StreamObserver<Empty> responseObserver) {
+    String workspacePath = getWorkspacePath(req.getWorkspaceName());
+
+    try {
+      fileUtils.deleteDirectory(workspacePath);
+      responseObserver.onNext(Empty.getDefaultInstance());
+      responseObserver.onCompleted();
+    } catch (IOException ignored) {
+      responseObserver.onError(
```

Comment:
Please add a comment:
`// We assume it's a missing workspace. That's the most probable reason.